azure-pipelines/azure-templates/helm-template.yml

```diff
@@ -1,41 +1,29 @@
 
 parameters:
-  - name: 'containerName'
-    type: string
-  - name: 'imageName'
-    type: string
-  - name: 'tag'
-    type: string
-  - name: 'activeSvc'
-    type: string
   - name: 'aksConnection'
     type: string
+    default: 'k8s-team3-config-dev'

```

Comment:
not really a good default. i don't know why everybody added so many defaults to the templates, when arguably the template files shouldn't be aware of things like this; pipelines shouldn't be able to depend on this default being what it is, they should have to specify what they want in their own files.
definitely not a blocking issue

Reply:
The default in particular is a testing one, but a pipeline will automatically fail(won't run at all) without a default for a k8s service connection. Since you're saying it shouldn't be in a template, is it better to just have it in the pipeline itself?

Reply:
it'll fail without a _value_, not without a default, no? it's up to the pipeline when it references the template to provide the parameter value. if no value is provided and there's no default then naturally there would be an error.